static_test_driver/static_test_driver.ino

```diff
@@ -235,40 +217,18 @@ void loop() {
   
   BEGIN_READ
   READ_FLAG(zero_force) {
-    #if CONFIGURATION == MK_2
     Serial.println(F("Zeroing load cell"));
     scale.tare(); // Load Cell, Assuming there is no weight on the scale, reset to 0
-
-    #else
-    if (force_zero_ready) {
-      Serial.println(F("Zeroing force plate"));
-      force_zero_ready = false;
-      force_zero_val = 0;
-      for (int i = 0; i < FORCE_PLATE_NUM_HIST_VALS; i++)
-        force_zero_val += force_hist_vals[i];
-      force_zero_val /= FORCE_PLATE_NUM_HIST_VALS;
-    }
-    else {
-      Serial.println(F("Force zero value not ready"));
-    }
-
-    #endif
   }
-  #if CONFIGURATION == MK_2
-  READ_FLAG(zero_pressure) {
-    if (pressure_zero_ready) {
-      Serial.println(F("Zeroing pressure"));
-      pressure_zero_ready = false;
-      pressure_zero_val = 0;
-      for (int i = 0; i < PRESSURE_NUM_HIST_VALS; i++)
-        pressure_zero_val += pressure_hist_vals[i];
-      pressure_zero_val /= PRESSURE_NUM_HIST_VALS;
-    }
-    else {
-      Serial.println(F("Pressure zero value not ready"));
-    }
+  
+  //TODO: Update flag names to match the following code
+  READ_FLAG(zero_pressure_fuel) {
+    zero_pressure (PRESSURE_FUEL, "fuel");
```

Comment:
Moving all this code into a seperate function doesn't actually simplify things that much - an important goal in software design is to maximize code reusability - having a function that takes in a pin number, uses that to find an array index, and then updates some global arrays based on that isn't very flexible or obvious what it does.  Duplicating code is sometimes OK, and often preferable when dealing with global variables.  

Instead, consider writing a function `float mean(unsigned count, float data[])` that computes the mean of a `float` array, and using that to simplify the existing zeroing code.  You can pass in the values of global variables, and use the result to update the appropriate global value.